README.md

```diff
@@ -86,8 +86,87 @@ Usage of ./hetty:
         CA private key filepath. Creates a new CA private key if file doesn't exist (default "~/.hetty/hetty_key.pem")
 ```
 
-⚠️ _Todo: Write instructions for installing CA certificate in local CA store, and_
-_configuring Hetty to be used as a proxy server._
+## Certificate Setup and Installation
+
+In order for Hetty to proxy requests going to HTTPS endpoints, certificates for 
+Hetty will need to be set up. Furthermore, these certificates may need to be
+installed to the host for them to be trusted by your browser. The following steps
+will cover how you can generate your certificate, provide them to hetty, and how
+you can install them in your local CA store.
+
+⚠️ _This process was done on a Linux machine but should_
+_provide guidance on Windows and macOS as well._
+
+### Generating CA certificates
+
+You can generate CA certificates two different ways. The first is bundled directly
+with hetty, and simplifies the process immensely. The alternative is using OpenSSL
+to generate them, which provides more control over expiration time and cryptography
+used, but requires you install the OpenSSL tooling. The first is suggested for any
+beginners trying to get started.
+
+#### Generating CA certificates with hetty
+
+Hetty will generate the default key and certificate on its own if none are supplied
+or found in `~/.hetty/` when first running the CLI. To generate a default key and
+certificate with hetty, simply run the command with no arguments
+
+```sh
+hetty
+```
+
+You should now have a key and certificate located at  `~/.hetty/hetty_key.pem` and
+`~/.hetty/hetty_cert.pem` respectively.
+
+#### Generating CA certificates with OpenSSL
+
+You can start off by generating a new key and CA certificate which will both expire
+after a month.
+
+```sh
+openssl req -newkey rsa:2048 -new -nodes -x509 -days 31 -keyout hetty_key.pem -out hetty_cert.pem
+```
+
+The default location which `hetty` will check for the key and CA certificate is under
+`~/.hetty/`, at `hetty_key.pem` and `hetty_cert.pem` respectively. You can move them
+here and `hetty` will detect them automatically. Otherwise, you can specify the
+location of these as arguments to `hetty`.
+
+```
+hetty -key key.pem -cert cert.pem
+```
+
+### Installing CA certificates
+
+In order for your browser to allow traffic to the local Hetty proxy, you may need
+to install these certificates to your local CA store. Otherwise, you may be met
+with errors related to HTTP Strict Transport Security (HSTS). In the real world this
```

Comment:
I don't this HSTS is relevant here in this context; this is about trust for the CA we've generated. We could maybe rephrase this paragraph to:

> Your browser first needs to trust the CA certificate we generated earlier. By default it doesn't, and any certificates signed by Hetty during proxying using the CA will be treated as invalid.

Reply:
The reason I included this was because I had tested this case myself in Firefox and ran into this error, and thought it may be useful for others who are confused why it's not working. I can removed this part if you would like.

Reply:
This has been removed as it may be misleading or irrelevant.